workshop/exercise-4/README.md

```diff
@@ -69,6 +69,30 @@ The node port in above sample output is `169.60.87.20:31702`
 while sleep 0.5; do curl http://<guestbook_endpoint/; done
 ```
 
+
+## Useful commands to generate load on the app
+
+In the following exercises, you'll use Istio to view load and requests on the app. You can use the provided shell snippits here to generate artificial load.
+
+To generate 404s
+
+```shell
+while true; do sleep 0.5;  curl http://<guestbook_endpoint>/lrandomurl; echo; done
+```
```

Comment:
is this /lrandomurl working?  I don't recall guestbook having this path exposed.  If not, I'd prefer a working example as many lab users like to c-n-p.